src/containers/Home/Home.jsx

```diff
@@ -1,60 +1,39 @@
-                                <div className="map">
-                                    {/* MAP WILL GO HERE  <p></p>*/}
-                                    <span>S t 0 K 3 - A  l  e  R  t</span>
-                                    <span>S t 0 K 3 - A  l  e  R  t</span>
-                                    <span>S t 0 K 3 - A  l  e  R  t</span>
-                                    <span>S t 0 K 3 - A  l  e  R  t</span>
-                                </div>
-                            </Segment>
-
-                            <Segment className="map-list-container" placeholder raised>
-                                <div>
-                                    {/* MAP LIST WILL GO HERE */}
-                                    <p>S t 0 K 3 - A  l  e  R  t</p>    
-                                </div>
-                            </Segment>
-                        </Segment>
-                    </Segment>    
-
-                    <div className="start-menu">
-                        <Dropdown text="Start" upward icon="chevron up" floating labeled button id="right-arrow" className="icon" > 
-                            <Dropdown.Menu>
-                                <Dropdown.Header icon='hand peace outline' content='Stoke_Alert_Menu_' />
-                                <Dropdown.Item>Important</Dropdown.Item>
-                                <Dropdown.Item>Announcement</Dropdown.Item>
-                                <Dropdown.Item icon='phone' content='Contact' subheader='Stoke_Alert_ Facebook page' />
-                                <Dropdown.Item icon='question' text='About' />
-                            </Dropdown.Menu>
-                        </Dropdown>
+state = {
+	tabBarName: localStorage.getItem('username'),
+}
+    
+tokenMatch = () => {
+	const { validations } = this.state;
+	let tokenStorage = localStorage.getItem('token');
+	if (validations.token != null && validations.token === tokenStorage) {
+		this.changeValidation('signedIn', true);
+	} else {
+		this.changeValidation('signedIn', false);
+	}
+}
 
-                        <Button className="start-menu-icon" icon>
-                            <Icon className="fb-icon"  name='facebook official' size='large' />
-                        </Button>
-                        <Button onClick={this.getUser} className="start-menu-icon-ig" icon>
-                            <span className="ig-icon">
-                                <Icon name='instagram' size='large'/>
-                            </span>
-                        </Button>
-                    </div>     
-                </Segment>   
-            </div>
-        )
-    }
+render() {
+	return (
+		<div className="wrapper-container">
+			<Segment className="div-thang" raised>
+				<TabBar 
+					tabMessage={`U S E R: ${this.state.tabBarName !== null ? this.state.tabBarName : ''}` }
+					name = 'M a p_'
+				/>
+				<HomeComponent />
+				<StartMenu />
+			</Segment>
+		</div>
+	);
+}
 }
```

Comment:
There's 2 closing brackets in the same alignment. The `render()` function needs to be spaced more towards the right.